releasenotes-dca/notes/merge-namespace-targets-4c82b02d2c1dc1ca.yaml

```diff
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    Customers using Single Step Instrumentation with target based workload selection can now use language detection.
+    Language detection greatly reduces startup time when all default libraries are configured for a target. To enable
+    this feature, set `language_detection.enabled` to `true`.
```

Comment:
`language_detection` is enabled by default so we don't need to add this here.

- https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L593-L597
- https://github.com/DataDog/datadog-operator/blob/main/api/datadoghq/v2alpha1/datadogagent_types.go#L239-L245